ssr-opengraph/src/index.tsx

```diff
@@ -41,11 +41,7 @@ app.get('/:chain/:type/:id/*', async (c) => {
   const type = c.req.param('type');
 
   if (chains.includes(chain)) {
-    if (type === 'collection') {
-      return await ogiRequest(c.req.url, c.req.raw.headers)
-    }
-
-    if (type === 'gallery' || type === 'detail') {
+    if (type === 'gallery' || type === 'detail' || type === 'collection') {
```

Comment:
type === 'u'?

Reply:
not yet, currently in progress `/blog` first, wdyt?

Reply:
> not yet, currently in progress `/blog` first, wdyt?

sometimes I forgot we do have a blog lol